pages/production.md

```diff
@@ -175,6 +175,17 @@ There are many other options that you can find in [Spring Boot documentation](ht
 
 When deploying a JHipster app to an Application Server or customizing your context-path, it is required to set the `base` value in `webpack.common.js`or `webpack.prod.js` equal to the expected context-path.
 
+In order to use the swagger api documentation site you need to adapt the swagger index file under `src/main/webapp/swagger-ui/index.html`. 
```

Comment:
We also have similar details under `Building an executable JAR / WAR file`. It would be good to club along that.

Reply:
Maybe we can add the snippet (also it is duplicated) for webpack configuration here too.

Reply:
I think the `Build` section is more appropriate for such customization and should remove all duplication from here. WDYT?

Reply:
Good point not sure what people are looking for. I will have a look and see if removing the dedicated running under context path makes sense eventually.